pkg/operator/ceph/csi/cluster_config.go

```diff
@@ -203,5 +205,7 @@ func SaveClusterConfig(
 	if _, err := clientset.CoreV1().ConfigMaps(csiNamespace).Update(configMap); err != nil {
 		return errors.Wrapf(err, "failed to update csi config map")
 	}
+
+	logger.Infof("successfully updated csi config map %q in namespace %q", configMap.Name, clusterNamespace)
```

Comment:
Will this update be at every interval, or is the configmap only updated when there is a change in mons? Just wanted to confirm if we're only logging this when it changed.

Reply:
Right, need to move this.